packages/babel-cli/src/babel/index.js

```diff
@@ -259,6 +259,7 @@ delete opts.configFile;
 delete opts.deleteDirOnStart;
 delete opts.keepFileExtension;
 delete opts.relative;
+delete opts.sourceMapTarget;
```

Comment:
There is still a `--source-map-target` CLI option that sets the `file` field above now when concatenating things into a single file.

Reply:
If you're using Babel 7, you absolutely need to use a lockfile. `grunt-babel` has not yet been updated to support Babel 7's beta. For now I would recommend locking your version of Babel and all the plugins to `beta.40`. `beta.41` is not compatible yet.

Reply:
`grunt-babel@8.0.0-beta.0` was released to support the scoped versions of babel including the latest version.